98 Commented/00 Intro/00 Boilerplate/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+/*
+WEBPACK is a module bundler for modern JavaScript applications. 
+When webpack processes your application, it recursively builds a dependency graph,
+that includes every module your application needs, then packages all of those modules into one or more bundles.
+
+There are a lot of statements that have been standardized in ES2015. 
+Although they are not supported in most browsers yet, webpack does support them out of the box.
+
+Webpack "transpiles" the code so that older browsers can also run it.
+*/
+
+module.exports = {
+    //An entry point indicates which module webpack should use to begin building out its internal dependency graph
+    // So, webpack will start reading on this entry to build our app, web, etc.
+  entry: ['./students.js'],
+    //The output property tells webpack where to emit the bundles it creates and how to name these files
+    // in this case, we are saing to wepack that built our app in the file: "bundle.js", and this will be
+    // the file served to clients how enter in our resources (our app, our web, etc. depending on our development)
+  output: {
+    filename: 'bundle.js',
+  },
+    // this section is for configuration regarding modules
+  module: {
+      // rules for modules (configure loaders, parser options, etc.)
+        // module.rules => allows you to specify several loaders within your webpack configuration. 
+        // This is a concise way to display loaders, and helps to maintain clean code
+        //Loaders enable webpack to process more than just JavaScript files (webpack itself only understands JavaScript).
+        //webpack loaders transform all types of files into modules that can be included in your application's dependency graph.
+    rules: [
+      // The next rule add support for es6, we will ask webpack to handle all js files under the project folder 
+      // (excluding the node_modules subfolder) and transpile them from es6 to es5 (using the babel-loader).
+      {
+          //test: Identify which file or files should be transformed by a certain loader
+          // We can use regular expressions, so here we are saying to webpack that process any files with extension starting in ".js_" with babe's loader
+        test: /\.js$/,
+          //with exclude we can say to webpack where don't proceess files to build our app/web.
+        exclude: /node_modules/,
+          //charge babel's loader for "transpilar" from ES6 to ES5
```

Comment:
Use _babel-loader_ to transpile from ES6 to ES5. We can use a single loader or we can chain the execution of several loaders.

Reply:
Ok, changes written and commit done with #comment5 (this changes has been made on the three projects that I had downloaded)